src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java

```diff
@@ -254,15 +297,15 @@ private boolean invokePathRead(final PathOperation<? extends PathItem> pathOp) {
 		}
 
 		try (final ClientFactory clientFactory = ClientFactory.withScope(scope, controllerURI);
-			 EventStreamReader<String> reader = clientFactory.createReader("reader",
-					 readerGroup,
-					 new JavaSerializer<String>(),
-					 ReaderConfig.builder().build())) {
+		     EventStreamReader<String> reader = clientFactory.createReader("reader",
+				     readerGroup,
+				     new JavaSerializer<String>(),
+				     ReaderConfig.builder().build())) {
 			System.out.format("Reading all the events from %s/%s%n", scope, streamName);
 			EventRead<String> event = null;
 			do {
 				try {
-					event = reader.readNextEvent(readerTimeoutMs);
+					event = reader.readNextEvent(Constants.READER_TIMEOUT_MS);
```

Comment:
I thought that the configuration option has been merged already. Why does the constant is used here instead?